common/djangoapps/third_party_auth/lti.py

```diff
@@ -44,12 +44,15 @@ def start(self):
         # Clean any partial pipeline data
         self.strategy.clean_partial_pipeline()
 
-        # Save validated LTI parameters (or None if invalid or not submitted)
-        validated_lti_params = self.get_validated_lti_params(self.strategy)
-
         # Set a auth_entry here so we don't have to receive that as a custom parameter
         self.strategy.session_setdefault('auth_entry', 'login')
 
+        # Save validated LTI parameters (or None if invalid or not submitted)
```

Comment:
Also, this docstring is no longer valid since `get_validated_lti_params` now raises an exception instead of returning `None`.